recipe/meta.yaml

```diff
@@ -21,9 +21,15 @@ requirements:
   host:
     - python
     - tensorflow-base {{ tensorflow }}
+    - typing {{ typing }}  # [py>=38]
+    - typing_extensions {{ typing }}  # [py<38]
+    - python-flatbuffers {{ flatbuffers }}
   run:
     - python
     - tensorflow-base {{ tensorflow }}
+    - typing {{ typing }}  # [py>=38]
+    - typing_extensions {{ typing }}  # [py<38]
```

Comment:
We need to change this pin to `typing_extensions` so that it is set to 3.7.4.* and not 3.7.4.1.* which is `typing` version. Without this change, validate_config fails for tensorflow environment.

Reply:
OK. And I think we agreed to remove `typing` altogether since it's included in py3 now.  I'll update.